kubeflow-pipelines/odh-ray-cluster/secret-generator.yaml

```diff
@@ -2,6 +2,6 @@ apiVersion: viaduct.ai/v1
 kind: ksops
 metadata:
   # Specify a name
-  name: aws-secret-generator
+  name: ceph_S3-secret-generator
```

Comment:
mixing between _ and - is visually jarring if uneeded fwiw

Reply:
secrets have a naming convention as well: only lowercase chars, ' - ' and ' . ' are allowed